evaluator.js

```diff
@@ -412,15 +410,15 @@ function get_args(arg_funcs, env, succeed, fail) {
         ? succeed(null, fail)
         : head(arg_funcs)(env,
                          // success continuation for this arg_func
-	                      (arg, fail2) => {
-		                     get_args(tail(arg_funcs),
+                        (arg, fail2) => {
```

Comment:
Should we keep these changes in a separate PR, as they're not related to the `const` declaration issue?
It will make it easier to revert changes if each PR is independent.

Reply:
Lines 129 and 156 are necessary.
There are no other changes except for a reduction in leading whitespaces for 2 of the lines.